src/styles.css

```diff
@@ -654,30 +654,36 @@ cg-container .cg-shapes {
 }
 
 square.last-move {
-	background: var(--lastMove) !important;
+	background-color: var(--lastMove) !important;
+	opacity: 0.4 !important;
+}
+
+square.selected {
+	background-color: var(--lastMove) !important;
 	opacity: 0.4 !important;
 }
 
 square.premove-dest {
 	background: radial-gradient(
 		var(--preMove) 19%,
-		rgba(0, 0, 0, 0) 20%
+		transparent 20%
 	) !important;
 	opacity: 0.8 !important;
 }
 
 square.oc.premove-dest {
 	background: radial-gradient(
-		transparent 0%,
-		transparent 79%,
-		var(--preMove) 80%
+		transparent 52%,
+		var(--preMove) 54%,
+		var(--preMove) 68%,
+		transparent 70%
```

Comment:
The circles are the same interface as chess.com. Also, annotations disappear once you click to make a move, so the circles will never be on screen when annotations are. Personally, I've always thought the circles looked better than the native Lichess design, but I can understand someone having the reverse opinion. Is it possible to have a option to change between the two styles?

Reply:
Yes, you're right, annotations and pre-moves are never shown at the same time anyway, but I'd still personally prefer them to be visually distinct.
Adding this as an option is of course possible. I'm also experimenting a bit with other style settings and trying to figure out the easiest way to incorporate them.
I'll definitely put it on the feature list and when I find a good way to use style settings, it will be included. But it may take a little while before it's ready to publish.